editor/components/block-drop-zone/index.js

```diff
@@ -53,10 +54,35 @@ function BlockDropZone( { index, isLocked, ...props } ) {
 		}
 	};
 
+	const reorderBlock = ( event, position ) => {
+		if ( index !== undefined && event.dataTransfer ) {
```

Comment:
We can avoid unnecessarily tabbing the rest of this function (harms readability) by inversing this condition and turning it into an early return:

```js
if ( index === undefined || ! event.dataTransfer ) {
	return;
}
```

Reply:
Definitely, much cleaner - will refactor this. :-)